From 1e1d758a6cc588cc91897168dc2bd157555b4bf8 Mon Sep 17 00:00:00 2001
From: LiangRongrong <Liangrongrong@mprc.pku.edu.cn>
Date: Tue, 22 Nov 2011 16:13:21 +0800
Subject: [PATCH 080/641] UniCore64: add a space in asm pgtable.h

Fixed checkpatch error:
space required after that ';' (ctx:VxV)
+#define pgoff_to_pte(x)                        __pte({BUG(); 0;})
                                                               ^
Signed-off-by: Liang Rongrong <liangrongrong@mprc.pku.edu.cn>
---
 arch/unicore64/include/asm/pgtable.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/unicore64/include/asm/pgtable.h b/arch/unicore64/include/asm/pgtable.h
index f29fc89..ea6a04f 100644
--- a/arch/unicore64/include/asm/pgtable.h
+++ b/arch/unicore64/include/asm/pgtable.h
@@ -137,7 +137,7 @@ extern void set_pte(pte_t *ptep, pte_t pteval);
 
 #define pte_file(pte)			(pte_val(pte) & UC64_L3_FILE)
 #define pte_to_pgoff(x)			({BUG(); 0; })
-#define pgoff_to_pte(x)			__pte({BUG(); 0;})
+#define pgoff_to_pte(x)			__pte({BUG(); 0; })
 
 #define PTE_FILE_MAX_BITS		36 /* FIXME */
 
-- 
1.7.9.5

